src/modules/visitor-space/components/VisitorSpaceSearchPage/VisitorSpaceSearchPage.tsx

```diff
@@ -235,13 +236,24 @@ const VisitorSpaceSearchPage: NextPage = () => {
 	 * Methods
 	 */
 
+	const prepareSearchValue = (value = '') => {
+		if (value.trim() && !query.search?.includes(value)) {
```

Comment:
also trim any whitespace before searching in query.search
```suggestion
		if (value.trim() && !query.search?.includes(value.trim())) {
```